index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Chocolate Pizza</title>
+</head>
+
+<div class="header_container">
+    <div id= "blogTitle">
+    <img src="https://github.com/alchemycodelab/alchemy-bootcamp-i-january-2019/blob/master/curriculum/week-02/class-09/LAB-10/LAB-10a-chocolate-pizza/lab-assets/logo.png?raw=true">
+    <div>
+    <P>Delicious</P>
+    <p>THE BEST FOOD BLOG ON THE WEB</p>
+    </div>
+    </div>
+    
+    <div id= "links">
+        <table>
+            <tr>
+                <td><img src= "https://github.com/alchemycodelab/alchemy-bootcamp-i-january-2019/blob/master/curriculum/week-02/class-09/LAB-10/LAB-10a-chocolate-pizza/lab-assets/fb-icon.png?raw=trueg"></img></td>
+                <td><img src= "https://github.com/alchemycodelab/alchemy-bootcamp-i-january-2019/blob/master/curriculum/week-02/class-09/LAB-10/LAB-10a-chocolate-pizza/lab-assets/twit-icon.png?raw=true"></img></td>
+                <td><img src= "https://github.com/alchemycodelab/alchemy-bootcamp-i-january-2019/blob/master/curriculum/week-02/class-09/LAB-10/LAB-10a-chocolate-pizza/lab-assets/gp-icon.png?raw=true"></img></td>
+                <td><img src= "https://github.com/alchemycodelab/alchemy-bootcamp-i-january-2019/blob/master/curriculum/week-02/class-09/LAB-10/LAB-10a-chocolate-pizza/lab-assets/insta-icon.png?raw=true"></img></td>
+                <td><img src= "https://github.com/alchemycodelab/alchemy-bootcamp-i-january-2019/blob/master/curriculum/week-02/class-09/LAB-10/LAB-10a-chocolate-pizza/lab-assets/flic-icon.png?raw=true"></img></td>
+                <td><img src= "https://github.com/alchemycodelab/alchemy-bootcamp-i-january-2019/blob/master/curriculum/week-02/class-09/LAB-10/LAB-10a-chocolate-pizza/lab-assets/pint-icon.png?raw=true"></img></td>
+                <td></td>
+                <td><img src= "https://github.com/alchemycodelab/alchemy-bootcamp-i-january-2019/blob/master/curriculum/week-02/class-09/LAB-10/LAB-10a-chocolate-pizza/lab-assets/rss-icon.png?raw=true"></img></td>
+               <td><img src= "https://github.com/alchemycodelab/alchemy-bootcamp-i-january-2019/blob/master/curriculum/week-02/class-09/LAB-10/LAB-10a-chocolate-pizza/lab-assets/mail-icon.png?raw=true"></img></td>
+               
+            </tr>
+        </table>
```

Comment:
You don't really need the table here. The best way to style these images would be to use flexbox on the parent div.